src/algebra/char_zero/defs.lean

```diff
@@ -0,0 +1,66 @@
+/-
```

Comment:
Can you do this file split as a separate PR, to shrink the diff a little?

Reply:
I can also split the nat/cast/defs and int/cast/defs splits if you think that would help.

Reply:
I think that would be a good idea, yes